endpoints/openrtb2/amp_auction_test.go

```diff
@@ -419,7 +421,8 @@ func TestOverrideWithParams(t *testing.T) {
 					Imp:  []openrtb2.Imp{{Banner: &openrtb2.Banner{Format: []openrtb2.Format{}}}},
 					Site: &openrtb2.Site{Ext: json.RawMessage(`{"amp":1}`)},
 				},
-				errorMsgs: nil,
+				errorMsgs:         nil,
+				expectFatalErrors: false,
```

Comment:
Nitpick: instead of explicitly calling out `false` for `expectFatalErrors` in these test cases, you could just omit this field and lean on the default value of `false` to cut down the size of the test.

Reply:
Removed